Moto/G6/res/values/config.xml

```diff
@@ -10,7 +10,7 @@
 	<integer name="config_brightness_ramp_rate_fast">200</integer>
 	<integer name="config_brightness_ramp_rate_slow">40</integer>
 	<bool name="config_setColorTransformAccelerated">true</bool>
-	<bool name="config_nightDisplayAvailable">false</bool>
```

Comment:
By default, config_nightDisplayAvailable is set to config_setColorTransformAccelerated
So simply removing this line will do

Reply:
Done. (dc8bf86)